src/main/java/baubles/common/network/PacketSyncBauble.java

```diff
@@ -60,20 +52,7 @@ public IMessage onMessage(PacketSyncBauble message, MessageContext ctx) {
 		if (world == null) return null;
 		Entity e = world.getEntityByID(message.playerId);
 		if (e instanceof EntityPlayer player) {
-			InventoryBaubles baubles = PlayerHandler.getPlayerBaubles(player);
-			if (message.initial) {
-				if (message.slot == 0) {
-					PlayerHandler.clearClientPlayerBaubles();
-					baubles = PlayerHandler.getPlayerBaubles(player);
-				}
-				baubles.stackList[message.slot] = message.bauble;
-				if (message.bauble != null && message.bauble.getItem() instanceof IBauble itemBauble) {
-					itemBauble.onPlayerLoad(message.bauble, player);
-				}
-			}
-			else {
-				baubles.setInventorySlotContents(message.slot, message.bauble);
-			}
+            PlayerHandler.getPlayerBaubles(player).stackList[message.slot] = message.bauble;
```

Comment:
Need to use `setInventorySlotContents` here, otherwise it won't sync to the client. You can test that by equipping a botania reach ring with right-click.